### PR TITLE
Allow for creation of controller without typed actions.

### PR DESCRIPTION
**Pull request by jeremyabbott**
This PR addresses the problem described in #38 where controllers with only parameterless actions would compile fine, and then crash at runtime.

### CODE REVIEW

**Comment by jeremyabbott** [comment]
WIP for SaturnFramework/Saturn#38

File: src/Saturn/Controller.fs [hunk 1]
```diff
@@ -42,22 +43,29 @@ module Controller =
...
+        | _ ->
+          match typeof<'Key> with
+          | k when k = typeof<bool> -> Bool
+          | k when k = typeof<char> -> Char
+          | k when k = typeof<string> -> String
+          | k when k = typeof<int32> -> Int32
+          | k when k = typeof<int64> -> Int64
+          | k when k = typeof<float> -> Float
+          | k when k = typeof<System.Guid> -> Guid
+          | k -> failwithf "Type %A is not a supported type for controller<'T>." k
```
Comment (forki) [inline src/Saturn/Controller.fs]: Please list all supported cases or link to the docs where this is specified

File: global.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "2.1.4"
```
Comment (forki) [inline global.json]: I think there are newer versions avaliable

Comment (jeremyabbott) [reply global.json]: @forki I went with what the build script is doing. Should I update build.fsx too? I know the SAFE template is using 2.1.105

File: src/Saturn/Controller.fs (hunk 1 above)
Comment (jeremyabbott) [reply src/Saturn/Controller.fs]: @forki I just pushed up a newer version that removes the weird `Obj` case, and lists out all supported types.

**Comment by Krzysztof-Cieslak** [comment]
LGTM, thanks a lot! :-)

**Pull request merged**